File: C/BEGINNER/1180 Lowest Number and Position.c
#include <stdio.h>

int main()
{
    int vectorSize;
    scanf("%d", &vectorSize);
    int vector[vectorSize], i, min, position;
    for (i = 0; i < vectorSize; i++)
        scanf("%d", &vector[i]);
    min = vector[0];
    position = 0;
    for (i = 1; i < vectorSize; i++)
    {
        if (vector[i] < min)
        {
            min = vector[i];
            position = i;
        }
    }
    printf("Menor valor: %d\n", min);
    printf("Posicao: %d\n", position);
    return 0;
}

File: Python/BEGINNER/1180 Lowest Number and Position.py
vector_size = int(input())
vector = list(map(int, input().split()))
smaller = vector[0]
position = 0

for i in range(1, len(vector)):
    if vector[i] < smaller:
        smaller = vector[i]
        position = i

print("Menor valor: %d" % smaller)
print("Posicao: %d" % (position))
